spec/features/users/guest_user_spec.rb

```diff
@@ -2,8 +2,8 @@
 
 RSpec.describe 'Guest User', type: :feature do
 
-  let!(:activity)      { create(:status_update, :published, team: team1) }
-  let!(:other_user)    { create(:user) }
+  let!(:activity) { create(:status_update, :published, team: team1) }
+  let(:other_user) { create(:user, hide_email: false) }
```

Comment:
Isn't this change unnecessary? Removing it does not make the spec fail (I assume `hide_email` defaults to `false`).

Are you doing it for clarity? 👍 if you are.